app/models/admin/project.rb

```diff
@@ -112,10 +112,11 @@ def create_or_update_default_project_from_settings_yml
       attributes = {
         :name => name,
         :url => url,
+        :jnlp_url => 'http://google.com',
```

Comment:
Does this also work with `nil` or `''`? If so, let's change this to one of those. If not, we should probably change this to an actually existing jnlp url: `http://jars.dev.concord.org/dev3/org/concord/maven-jnlp/itsisu-otrunk/itsisu-otrunk-0.1.0-20121127.152823-c.jnlp` is the one ITSI currently uses.